changelogs/fragments/275-add_ignore_txt.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
```

Comment:
```suggestion
---
trivial:
```

Reply:
Without the start of document directive, CI will fail.

Reply:
committed, thanks